Gemfile

```diff
@@ -9,12 +9,12 @@ gem 'solidus', github: 'solidusio/solidus', branch: branch
 # Needed to help Bundler figure out how to resolve dependencies,
 # otherwise it takes forever to resolve them.
 # See https://github.com/bundler/bundler/issues/6677
-gem 'rails', '>0.a'
+gem 'rails', ENV.fetch('RAILS_VERSION', '>0.a')
```

Comment:
We're allowing the Rails version to be customized in development so that we can test the extension against both old Solidus versions which depend on Rails 6 and the latest Solidus version which depends on Rails 7.

Reply:
I don't think we need that fix for bundler anymore. Maybe we can take advantage of this change to also remove it and add the explicit Rails dependency only when specified with a ENV variable?